scripts/vercel.sh

```diff
@@ -77,10 +76,7 @@ function remap_database_integration() {
   if [[ "$NEXT_PRIVATE_DATABASE_URL" == *"neon.tech"* ]]; then
     log "Remapping for Neon integration"
 
-    PROJECT_ID="$(echo "$PGHOST" | cut -d'.' -f1)"
-    PGBOUNCER_HOST="$(echo "$PGHOST" | sed "s/${PROJECT_ID}/${PROJECT_ID}-pooler/")"
-
-    export NEXT_PRIVATE_DATABASE_URL="postgres://${PGUSER}:${PGPASSWORD}@${PGBOUNCER_HOST}/${PGDATABASE}?pgbouncer=true"
+    export NEXT_PRIVATE_DATABASE_URL="postgres://${PGUSER}:${PGPASSWORD}@${PGHOST}/${PGDATABASE}?pgbouncer=true"
```

Comment:
Now we need to account for the direct database url so prisma migrations can run 🥲

Reply:
Also we have a repeat of the logic in `packages/prisma/helper.ts`

Reply:
> Now we need to account for the direct database url so prisma migrations can run 🥲

Apparently if we upgrade prisma to 5.10 it lets us migrate using a connection pool?

https://neon.tech/docs/guides/prisma#connection-pooling-with-prisma-migrate

> Also we have a repeat of the logic in packages/prisma/helper.ts

The logic in the helper checks whether pooler is already attached, want me to try clean it up anyway?

```
  // Support for neon.tech (Neon Database)
  if (url.hostname.endsWith('neon.tech')) {
    const [projectId, ...rest] = url.hostname.split('.');

    if (!projectId.endsWith('-pooler')) {
      url.hostname = `${projectId}-pooler.${rest.join('.')}`;
    }

    url.searchParams.set('pgbouncer', 'true');

    process.env.NEXT_PRIVATE_DATABASE_URL = url.toString().replace('https://', 'postgres://');
  }
```

Reply:
Also what was the reason we can't use the direct database URL for neon for migrations?